### DISCUSSION TITLE
Fine tuned the open source seq2seq models

### DISCUSSION THREAD

**Discussion by Chetan-Yeola** [hpcaitech/ColossalAI#3434] [Community | Q&A]
Hello Team,
Can I use Colossalai to fine-tune the Text-to-Text generation model on a custom dataset? If so, please describe the procedures.

**Comment by binmakeswell** [comment]
Hi @Chetan-Yeola Yes, it accelerates training speed and reduces hardware resource requirements without limiting your model and dataset. You can refer to our examples:
https://github.com/hpcaitech/ColossalAI/tree/main/examples
https://github.com/hpcaitech/ColossalAI/tree/main/applications/Chat
If you need customized in-depth cooperation or support, please send the details to ybl@hpcaitech.com
